graylog2-server/src/main/java/org/graylog2/alerts/Alert.java

```diff
@@ -16,18 +16,28 @@
  */
 package org.graylog2.alerts;
 
-import org.graylog2.plugin.database.Persisted;
 import org.joda.time.DateTime;
 
 import java.util.Map;
 
 /**
  * @author Dennis Oelkers <dennis@torch.sh>
  */
-public interface Alert extends Persisted {
+public interface Alert {
+    String getId();
```

Comment:
Unfortunately Auto Value will fail to generate the builder class if it's missing:

```
Error:(90, 12) java: Expected a method with this signature: org.graylog2.alerts.AlertImpl.Builder id(java.lang.String)
```